### PR TITLE
added prerelease and release section for 1/25

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

File: microsoft-edge/webview2/releasenotes.md [hunk 1]
```diff
@@ -16,6 +16,51 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
 > [!NOTE]
 > Re-compile your app after updating the NuGet package.  
 
+## 1.0.774-prerelease
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:19]: Prerelease candidate is likely 773.
```suggestion
## 1.0.773-prerelease  
```

File: microsoft-edge/webview2/releasenotes.md [hunk 2]
```diff
@@ -16,6 +16,51 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
...
+#### General  
+
+> [!IMPORTANT]
+>  **Breaking Change**: WebView2 pre-release package 0.9.430 has been deprecated, and will be removed the upcoming release. Please discontinue development with this packages.  
+
+###### Features  
+*   Disabled Edge Shopping feature in WebView2. 
+*   Added [TrySuspend & Resume][ReferenceWin32Icorewebview210774PreReleaseTrySuspendResume] method that lets you suspend and resume a WebView. 
+*   Added [SetVirtualHostNameToFolderMapping][ReferenceWin32Icorewebview210774PreReleaseSetVirtualHostNameToFolderMapping] method that maps between a virtual host name and a folder path. 
+*   Added [DefaultBackgroundColor][ReferenceWin32Icorewebview2controllerViewWebview210774PreReleaseDefaultBackgroundColor] property that lets you set the background color and transparency.   \([\#414][GithubMicrosoftedgeWebviewfeedbackIssue549]\)
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:34]: Versions and link definitions to be added/updated.
```suggestion
*   Added [DefaultBackgroundColor][ReferenceWin32Icorewebview2controllerViewWebview210774PreReleaseDefaultBackgroundColor] property that lets you set the background color and transparency.   \([\#414][GithubMicrosoftedgeWebviewfeedbackIssue414]\)
```

File: microsoft-edge/webview2/releasenotes.md [hunk 3]
```diff
@@ -16,6 +16,51 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
...
+*   Added [UserAgent][ReferenceWin32Icorewebview2experimentalsettings10774PreReleaseGetUserAgent] property that lets you get and set the User Agent.\([\#122][GithubMicrosoftedgeWebviewfeedbackIssue549]\)
+
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:36]: Can you please add one more note?
```suggestion
*   Updated CreateCookieWithCookie to become CopyCookie.
```

File: microsoft-edge/webview2/releasenotes.md [hunk 4]
```diff
@@ -16,6 +16,51 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
...
+    * Visual Hosting APIs.
+    * URI Protocol Scheme.
+
+#### .NET  
+
+###### Bug fixes
+* Fixed bug in WPF that was crashing webview on closing window with F4.  \([\#399][GithubMicrosoftedgeWebviewfeedbackIssue399]\)
+* Fixed WebView2 initialization screen to be transparent, instead of gray.   \([\#196][GithubMicrosoftedgeWebviewfeedbackIssue196]\)
+
+## 1.0.705.51
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:58]: Release candidate is likely 705.50.

**Review by peiche-jessica — CHANGES_REQUESTED** [review]
There are a few bad markdown links that need to be fixed
- `[Webview2ConceptsDistributionUnderstandRuntimeInstallerPreview]` under 0.9.579 should become `[Webview2ConceptsDistributionUnderstandRuntimeInstaller]`
- `[ReferenceWin32IdlCreatecorewebview2environmentwithoptions09538` under 0.9.538 is missing a closing `]`

File: microsoft-edge/webview2/releasenotes.md (hunk 2 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/releasenotes.md]: `GithubMicrosoftedgeWebviewfeedbackIssue414` is undefined.

File: microsoft-edge/webview2/releasenotes.md (hunk 1 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/releasenotes.md]: Other version instances have to be updated too.

File: microsoft-edge/webview2/releasenotes.md [hunk 5]
```diff
@@ -16,6 +16,10 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
 > [!NOTE]
 > Re-compile your app after updating the NuGet package.  
 
+
+> [!NOTE]
+> Please note Minimum Edge version is for supporting running WebView. Some APIs are defined in newer interfaces. Please ensure you are using the proper runtime version for prerelease package APIs. You may also use Microsoft Edge Canary channel to ensure the APIs are supported.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:21]:
```suggestion
> Please note Minimum Edge version is for supporting running WebView. Some APIs are defined in newer interfaces. Please ensure you are using the proper runtime version for experimental APIs in prerelease packages. You may also use Microsoft Edge Canary channel to ensure the APIs are supported.
```

File: microsoft-edge/webview2/releasenotes.md [hunk 6]
```diff
@@ -47,7 +51,10 @@ Release Date: January 25, 2021
 ###### Promotions  
 *   Promoted the following experimental changes to stable:
     * Visual Hosting APIs.
-    * URI Protocol Scheme.
+    * [SetVirtualHostNameToFolderMapping][ReferenceWin32Icorewebview210774PreReleaseSetVirtualHostNameToFolderMapping]
+    * [TrySuspend & Resume][ReferenceWin32Icorewebview210774PreReleaseTrySuspendResume]
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:55]: There are no experimental versions of TrySuspend & Resume and DefaultBackgroundColor. They are fast-tracked to public IDL directly.

**Review by peiche-jessica — DISMISSED** [review]

File: microsoft-edge/webview2/releasenotes.md (hunk 1 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/releasenotes.md]: Changed to 773!

File: microsoft-edge/webview2/releasenotes.md (hunk 2 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/releasenotes.md]: https://github.com/MicrosoftEdge/WebView2Feedback/issues/414 it works for me?

File: microsoft-edge/webview2/releasenotes.md (hunk 4 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/releasenotes.md:66]: changed to 705.50!

File: microsoft-edge/webview2/releasenotes.md (hunk 6 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/releasenotes.md]: Yes, but they are stable in the prerelease, so I noted it as a 'promotion', even though it wasn't experimental.  Will add a note.

File: microsoft-edge/webview2/releasenotes.md [hunk 7]
```diff
@@ -16,6 +16,70 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
...
+
+> [!IMPORTANT]
+> **Breaking Change**:  WebView2 pre-release package 0.9.430 is deprecated, and is being removed the upcoming release.  Discontinue development with it.  
+
+###### Features  
+
+*   Added [TrySuspend and Resume][ReferenceWin32Icorewebview210773PreReleaseTrySuspendResume] method to suspend and resume a WebView.  
+*   Added [SetVirtualHostNameToFolderMapping][ReferenceWin32Icorewebview210773PreReleaseSetVirtualHostNameToFolderMapping] method that maps between a virtual host name and a directory path.  
+*   Added [DefaultBackgroundColor][ReferenceWin32Icorewebview2controllerViewWebview210773PreReleaseDefaultBackgroundColor] property to set the background color and transparency.  \([\#414][GithubMicrosoftedgeWebviewfeedbackIssue414]\)  
+*   Added [UserAgent][ReferenceWin32Icorewebview2experimentalsettings10773PreReleaseGetUserAgent] property to get and set the User Agent.\([\#122][GithubMicrosoftedgeWebviewfeedbackIssue549]\)
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:38]:
```suggestion
*   Added [UserAgent][ReferenceWin32Icorewebview2experimentalsettings10773PreReleaseGetUserAgent] property to get and set the User Agent.\([\#122][GithubMicrosoftedgeWebviewfeedbackIssue122]\)
```

File: microsoft-edge/webview2/releasenotes.md [hunk 8]
```diff
@@ -598,3 +657,16 @@ Initial developer preview release.
...
+[ReferenceWin32Icorewebview210773PreReleaseTrySuspendResume]: /microsoft-edge/webview2/reference/win32/icorewebview2_3?view=webview2-1.0.773-prerelease&preserve-view=true#TrySuspend "TrySuspend - interface ICoreWebview2_3 | Microsoft Docs"
+[ReferenceWin32Icorewebview2experimentalsettings10773PreReleaseGetUserAgent]: /microsoft-edge/webview2/reference/win32/icorewebview2/icorewebview2experimentalsettingsget_useragent "get_UserAgent - interface ICoreWebView2ExperimentalSettings | Microsoft Docs"
+[ReferenceWin32Icorewebview210773PreReleaseSetVirtualHostNameToFolderMapping]: /microsoft-edge/webview2/reference/win32/icorewebview2_3?view=webview2-1.0.773-prerelease&preserve-view=true#SetVirtualHostNameToFolderMapping "SetVirtualHostNameToFolderMapping - interface ICoreWebView2_3 | Microsoft Docs" 
+[ReferenceWin32Icorewebview2controllerViewWebview210773PreReleaseDefaultBackgroundColor]: //microsoft-edge/webview2/reference/win32/icorewebview2controller?view=webview2-1.0.773-prerelease&preserve-view=true#defaultbackgroundcolor "DefaultBackgroundColor - interface ICoreWebView2Controller2 | Microsoft Docs"
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:671]:
```suggestion
[ReferenceWin32Icorewebview2controllerViewWebview210773PreReleaseDefaultBackgroundColor]: /microsoft-edge/webview2/reference/win32/icorewebview2controller?view=webview2-1.0.773-prerelease&preserve-view=true#defaultbackgroundcolor "DefaultBackgroundColor - interface ICoreWebView2Controller2 | Microsoft Docs"
```

File: microsoft-edge/webview2/releasenotes.md [hunk 9]
```diff
@@ -598,3 +657,16 @@ Initial developer preview release.
...
+[ReferenceWin32Icorewebview2controllerViewWebview210773CompositionController]:  //microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller?view=webview2-1.0.773-prerelease&preserve-view=true
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:672]:
```suggestion
[ReferenceWin32Icorewebview2controllerViewWebview210773CompositionController]:  /microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller?view=webview2-1.0.773-prerelease&preserve-view=true
```

File: microsoft-edge/webview2/releasenotes.md [hunk 10]
```diff
@@ -598,3 +657,16 @@ Initial developer preview release.
 [DOMContentLoadedAPI]: /microsoft-edge/webview2/reference/win32/icorewebview2_2?view=webview2-1.0.721-prerelease#add_domcontentloaded&preserve-view=true "add_DOMContentLoaded - interface ICoreWebView2_2 | Microsoft Docs"
 [WebViewEnvironmentproperty]: /microsoft-edge/webview2/reference/win32/icorewebview2_2?view=webview2-1.0.721-prerelease#get_environment&preserve-view=true "ICoreWebView2CookieManager | Microsoft Docs"
 
+[GithubMicrosoftedgeWebviewfeedbackIssue196]:  https://github.com/MicrosoftEdge/WebViewFeedback/issues/196 "Feedback repo for MicrosoftEdge/WebViewFeedback Issue 196"
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:660]: Missing GithubMicrosoftedgeWebviewfeedbackIssue122

File: microsoft-edge/webview2/releasenotes.md [hunk 11]
```diff
@@ -16,6 +16,70 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
 > [!NOTE]
 > Re-compile your app after updating the NuGet package.  
 
+> [!NOTE]
+> The minimum version of Microsoft Edge supports running WebView.  Some APIs are defined in newer interfaces.  Ensure you use the proper runtime version for exerimental APIs in prerelease packages.  To ensure the APIs are supported, you may also use Microsoft Edge Canary channel.
```
Comment (jm-trd-ms) [inline microsoft-edge/webview2/releasenotes.md:20]:
```suggestion
> The minimum version of Microsoft Edge supports running WebView.  Some APIs are defined in newer interfaces.  Ensure you use the proper runtime version for experimental APIs in prerelease packages.  To ensure the APIs are supported, you may also use Microsoft Edge Canary channel.  
```

**Review by jm-trd-ms — CHANGES_REQUESTED** [review]

**Review by Reezaali — CHANGES_REQUESTED** [review]
Please review the feedback. Thank you.

File: microsoft-edge/webview2/releasenotes.md [hunk 12]
```diff
@@ -14,18 +14,80 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edg
...
+    
+##### Promotions  
+
+*   Promoted the following experimental changes to stable.  
+    *   Visual Hosting APIs.  
+    *   [SetVirtualHostNameToFolderMapping][ReferenceWin32Icorewebview210773PreReleaseSetVirtualHostNameToFolderMapping]  
+    *   [TrySuspend and Resume][ReferenceWin32Icorewebview210773PreReleaseTrySuspendResume] 
+    *   [DefaultBackgroundColor][ReferenceWin32Icorewebview2controllerViewWebview210773PreReleaseDefaultBackgroundColor]  
+    > [!NOTE]
+    > TrySuspend and Resume and DefaultBackgroundColor are APIs that are fast-tracked into stable. There are no experimental versions of these APIs.
```
[the remark this answers is not in the document]
Comment (jasonstephen15) [reply microsoft-edge/webview2/releasenotes.md:57]: Maybe unclear, these APIs are fast-tracked, which means they did not go through the whole 6weeks in experimental process. Developers may need to know this, even though it is unlikely

**Comment by jasonstephen15** [comment]
#sign-off

File: microsoft-edge/webview2/releasenotes.md [hunk 13]
```diff
@@ -14,18 +14,78 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edg
...
+
+    
+##### Bug fixes  
+
+*   Turned off the Microsoft Edge Shopping feature in WebView2.  
+*   Turned off the context menu in the PDF viewer when `AreDefaultContextMenusEnabled` is `false`.  \([\#605][GithubMicrosoftedgeWebviewfeedbackIssue605]\).  
+*   Fixed a bug that returned `E_NOINTERFACE` when querying `ICoreWebView2` for `ICoreWebView2Experimental`.  \([\#691][GithubMicrosoftedgeWebviewfeedbackIssue691]\).  
+*   Fixed a bug that allowed navigation with malformed URIs when `CoreWebView2NavigationStartingEventArgs.Cancel` was set to `false`.  \([\#400][GithubMicrosoftedgeWebviewfeedbackIssue400]\).  
+*   Fixed a bug that blocked `window.print()` on pop-up windows with event handlers attached to `NewWindowRequested` events.  \([\#409][GithubMicrosoftedgeWebviewfeedbackIssue409]\).  
+*   Fixed Dynamic DPI issue when moving apps between different monitors. \([\#58][GithubMicrosoftedgeWebviewfeedbackIssue549]\)
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:47]: A definition for `GithubMicrosoftedgeWebviewfeedbackIssue58` is required too.
```suggestion
*   Fixed Dynamic DPI issue when moving apps between different monitors. \([\#58][GithubMicrosoftedgeWebviewfeedbackIssue58]\)  
```

Comment (peiche-jessica) [reply microsoft-edge/webview2/releasenotes.md]: A definition for `GithubMicrosoftedgeWebviewfeedbackIssue58` is missing and `[ReferenceWin32IdlCreatecorewebview2environmentwithoptions09538` under 0.9.538 is missing a closing `]`

**Comment by jasonstephen15** [comment]
#sign-off @jm-trd-ms

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**